MonoGame.Framework/Graphics/GraphicsExtensions.cs

```diff
@@ -597,131 +621,113 @@ internal static int GetSwapInterval(this PresentInterval interval)
             // HdrBlendable implemented as HalfVector4 (see http://blogs.msdn.com/b/shawnhar/archive/2010/07/09/surfaceformat-hdrblendable.aspx)
             case SurfaceFormat.HdrBlendable:
             case SurfaceFormat.HalfVector4:
+                if (!supportsHalfFloat)
+                    goto case InvalidFormat;
                 glInternalFormat = PixelInternalFormat.Rgba16f;
                 glFormat = PixelFormat.Rgba;
                 glType = PixelType.HalfFloat;
                 break;
 
             case SurfaceFormat.HalfSingle:
+                if (!supportsHalfFloat)
+                    goto case InvalidFormat;

```

Comment:
@Jjagg not sure what is the best way to support this since it depends on the runtime  GL version. GraphicsExtensions is for the most part does not care about gl version. 

My first thought is to export a HalfSingleInternalPixelFormat property which we set appropriately on the GraphicsCapabilities. But that doesn't feel right.

Reply:
I think it's fine to check `GL.BoundAPI` and `GraphicsDevice.glMajorVersion`. I agree stuff like this shouldn't be in `GraphicsCapabilities`.